c/blake3_impl.h

```diff
@@ -45,6 +45,10 @@ enum blake3_flags {
 #include <immintrin.h>
 #endif
 
+#if defined(__aarch64__) && !defined(BLAKE3_NO_NEON) && !defined(BLAKE3_USE_NEON)
```

Comment:
Do we need to test `_M_ARM64` here in addition to `__aarch64__`, to cover MSVC?

Reply:
Or maybe it would make sense to factor that into a separate variable, like we've done with `#if defined(__x86_64__) || defined(_M_X64) ` above?